tajo-core-tests/src/test/resources/results/TestGroupByQuery/testDistinctAggregation_case10.result

```diff
@@ -1,3 +1,3 @@
 ?sum,?sum_1
 -------------------------------
-3,414440.9
\ No newline at end of file
+3,414440.89999999997
```

Comment:
I tested in IDE debug tool
The precision is changed in SumContext and OffheapWriter keeps number with precision.